main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race: dict) -> Race | bool:
```

Comment:
The return type annotation is incorrect, you indicate that you return with a `Race` class instance, or a boolean value. Actually, you return a tuple of these objects. Please, review your return type annotation or omit these methods and call these actions in the code directly